### DISCUSSION TITLE
Git Desktop issues

### DISCUSSION THREAD

**Discussion by fetbarcelon** [community/community#63301] [Actions]
### Select Topic Area

Bug

### Body

I seem to be getting this issue trying to push stuff using Git Desktop with a large Unity project with LFS enabled and with a large Unreal Engine project. 
. I can't get a single commit to work.
. Once it finishes the Writing I get a popup saying
The remote disconnected. Check your Internet connection and try again.! 
My internet works well, IDK how to fix it. 

2.) Another issues is the Git that can't install. The Command Line Prompt supposed was install, but when I go to Git Desktop, it shows that it is not installed.

Any solutions?
Can someone help me? Please

Thanks

**Comment by diegocatao** [comment]
Hi @fetbarcelon,
What's your OS?

**Comment by fetbarcelon** [comment]
Not I am Windows 10Sent from my iPhoneOn Aug 9, 2023, at 4:10 PM, Catão ***@***.***> wrote:﻿
Hi @fetbarcelon,
What's your OS?

**Comment by steveward** [comment]
Could you upload the log file from GitHub Desktop so that I could get some more information about this error? To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs`. The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. 

That should show why the push is timing out.
